corehq/apps/domain/templates/login_and_password/two_factor/_base.html

```diff
@@ -0,0 +1,7 @@
+{% extends "style/bootstrap2/centered.html" %}
```

Comment:
Assuming this was done before we started pushing harder to get everything on B3... do you have bandwidth to migrate this once it's merged?